src/ContainerAwareCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Webmozarts Console Parallelization package.
+ *
+ * (c) Webmozarts GmbH <office@webmozarts.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Webmozarts\Console\Parallelization;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+abstract class ContainerAwareCommand extends Command implements ContainerAwareInterface
```

Comment:
Why is this class needed?

Reply:
The `ContainerAwareCommand` class is deprecated since Symfony 4.2 and removed since Symfony 5.0. It was used in the past to create commands extending from it so they had direct access to the app service container. The alternative is to extend commands from the `Command` class and use proper service injection in the command constructor.
However, I recreated this class to maintain the current behavior.

https://symfony.com/blog/new-in-symfony-4-2-important-deprecations#deprecated-containerawarecommand